UI/Shared/GameGrid.razor

```diff
@@ -0,0 +1,32 @@
+@using TicTacGo.Enums
+
+<div class="bg-indigo-600 grid grid-cols-3 grid-rows-3 gap-2 h-80 w-80 m-auto mt-2 mb-2 p-2 rounded-lg">
+    @for (int i = 0; i < GridCells.Length; i++)
+    {
+        Marks cell = GridCells[i];
+        int Index = i;
+        <button @onclick="((e)=> HandleMark(cell, Index))" class="bg-indigo-900 rounded-lg  flex items-center justify-center cursor-pointer">
+           @if (cell.ToString() == "NotSet")
+           {
+            
+           }else if (cell.ToString() == "X"){
```

Comment:
Fix spacing, at the start and at the end. Also, compare the enums, you do not need to call `.ToString()`. See the example I gave in your HandleMark function.